database/db_models.py

```diff
@@ -151,8 +152,18 @@ class TokenTransaction(db.Model):
     def __str__(self):
         return '%s' % (self.name)
 
-class CirculatingSupply(db.Model):
-    __tablename__ = 'circulating_supply'
+class CirculatingOgnSupply(db.Model):
```

Comment:
A migration file needs to be added in order for the new table to get created.
See https://github.com/OriginProtocol/origin-website/tree/master/migrations

See flask-migrate doc regrding how to generate a new migration file
https://flask-migrate.readthedocs.io/en/latest/

The migration runs automatically as part of the deploy
See https://github.com/OriginProtocol/origin-website/blob/087dc8aaa287f247d764f8544a5f69aa22163a9b/Procfile#L3